src/components/atoms/InternalTooltip/InternalTooltip.tsx

```diff
@@ -66,76 +66,31 @@ const StyledSvg = styled.svg<StyledSvgProps>`
  * A primitive tooltip to be used as a basis for more opinionated UI components.
  */
 export const InternalTooltip = ({
-  isOpen,
   children,
-  tooltip,
   $background = "black",
   $color = "text-inverted",
   tooltipPosition = "bottom-left",
   ...props
 }: InternalTooltipProps) => {
-  const positionProps = (() => {
-    const props: Partial<OakFlexProps> = {};
-
-    switch (tooltipPosition) {
-      case "top-left":
-      case "top-right":
-        props.$top = "space-between-none";
-        props.$transform = `translateY(calc(-100% - ${parseSpacing(
-          "space-between-s",
-        )}))`;
-        break;
-      default:
-        props.$bottom = "space-between-none";
-        props.$transform = `translateY(calc(100% + ${parseSpacing(
-          "space-between-s",
-        )}))`;
-        break;
-    }
-
-    switch (tooltipPosition) {
-      case "top-left":
-      case "bottom-left":
-        props.$left = "space-between-none";
-        break;
-      default:
-        props.$right = "space-between-none";
-        break;
-    }
-
-    return props;
-  })();
-
```

Comment:
I decided that this was too much implementation for `InternalTooltip` — It is now only responsible for drawing the tooltip box and positioning the arrow/tail to the correct corner. `OakTooltip` is now responsible for positioning and the visibility state.